src/runtime_src/core/pcie/driver/linux/xocl/subdev/qdma.c

```diff
@@ -1139,7 +1139,7 @@ static ssize_t queue_rw(struct xocl_qdma *qdma, struct qdma_stream_queue *queue,
 		spin_unlock_bh(&queue->req_lock);
 	}
 
-	return kiocb ? -EIOCBQUEUED : 0;
+	return -EIOCBQUEUED;
```

Comment:
since, if kiocb is null then if() at 1120 line number will be true, hence control won't reach here as there is a return in if(){}.